lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -919,7 +919,8 @@ def test_accordion(self):
         )
 
         # removes newlines and whitespace from the returned view string
-        view = ''.join(render_accordion(request, self.course, table_of_contents['chapters']).split())
+        # add language preference, default is fine.
```

Comment:
Does this comment seem like it would still be valid after reading it in a month?